indra/assemblers/html/assembler.py

```diff
@@ -23,25 +26,78 @@
     make_top_level_label_from_names_key, make_stmt_from_relation_key, \
     reader_sources, db_sources, all_sources, get_available_source_counts, \
     get_available_ev_counts, standardize_counts, get_available_beliefs, \
-    StmtGroup, make_standard_stats
+    StmtGroup, make_standard_stats, internal_source_mappings, \
+    available_sources_stmts, available_sources_src_counts
 from indra.literature import id_lookup
 
 logger = logging.getLogger(__name__)
 HERE = dirname(abspath(__file__))
 
+# Derived types
+SourceInfo = Dict[str, Dict[str, Union[str, Dict[str, str]]]]
+SourceColors = List[Tuple[str, Dict[str, Union[str, Dict[str, str]]]]]
+
 
 loader = FileSystemLoader(join(HERE, 'templates'))
 env = Environment(loader=loader)
 
 default_template = env.get_template('indra/statements_view.html')
 
+reader_order = ['reach', 'sparser', 'medscan', 'trips', 'eidos']
+
+
+def _sort_readers(reader_list: List[str]):
+    """Sort readers according to reader_order"""
+    reader_list.sort(key=lambda r: reader_order.index(r)
```

Comment:
`key=lambda r: reader_order.index(r)` is the same as just `key=reader_order.index`

Reply:
`reader_order` sometimes is only a subset of the provided readers in `reader_list` and errors in that case:
```
In [8]: lst = [8, 23, 99, 7, 9, 0]

In [9]: sub_lst = [8, 0,  9]

In [10]: lst.sort(key=sub_lst.index)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-14-7f03eae0e9ed> in <module>()
----> 1 lst.sort(key=sub_lst.index)

ValueError: 23 is not in list
```

Reply:
Wouldn't you get the same error when wrapping it with a lambda?

Reply:
The following if statement handles the error

Reply:
:point_up:

Reply:
bahhh I'm really sorry I totally didn't see that